homeassistant/components/bmw_connected_drive/coordinator.py

```diff
@@ -53,6 +54,7 @@ def __init__(self, hass: HomeAssistant, *, entry: ConfigEntry) -> None:
         super().__init__(
             hass,
             _LOGGER,
+            config_entry=entry,
```

Comment:
```suggestion
            config_entry=config_entry,
```

Reply:
Ah, sorry. That's what you meant!

Reply:
PR gets bigger and bigger :)
Had to move `type BMWConfigEntry` to `coordinator.py` due to otherwise circular imports.